src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
```

Comment:
The `slice` method is used incorrectly here. Since `fromFormat` and `toFormat` are arrays, and the separator is the 4th element (index 3), you should be accessing it using index notation, not `slice`. Use `fromFormat[3]` instead of `fromFormat.slice(3)`.